.github/scripts/trymerge.py

```diff
@@ -1507,6 +1507,34 @@ def checks_to_markdown_bullets(
     ]
 
 
+def post_starting_merge_comment(
+    repo: GitRepo,
+    pr: GitHubPR,
+    explainer: TryMergeExplainer,
+    dry_run: bool,
+    ignore_current_checks_info: Optional[
+        list[tuple[str, Optional[str], Optional[int]]]
+    ] = None,
+) -> None:
+    """Post the initial merge starting message on the PR. Also post a short
+    message on all PRs in the stack."""
+    gh_post_pr_comment(
+        pr.org,
+        pr.project,
+        pr.pr_num,
+        explainer.get_merge_message(ignore_current_checks_info),
+        dry_run=dry_run,
+    )
+    for additional_prs, _ in get_ghstack_prs(repo, pr):
```

Comment:
Bug: As per the comment [on this function](https://github.com/pytorch/pytorch/blob/67e97fcd48b428ce8c7ff9bb118159b540f169b1/.github/scripts/trymerge.py#L652), this will also return the original PR along with all the PRs below it in the stack, meaning that the PR that the merge command was made on will get double comments

Reply:
Added check that the pr numbers are not the same